context.go

```diff
@@ -179,6 +179,10 @@ func (c *Ctx) resolveProjectRoot(path string) (string, error) {
 //
 // The second returned string indicates which GOPATH value was used.
 func (c *Ctx) SplitAbsoluteProjectRoot(path string) (string, error) {
+	if path == filepath.Join(c.GOPATH, "src") {
+		return "", errors.Errorf("Initializing a project in $GOPATH/src directly is not supported currently")
```

Comment:
just nits :smile:

1. use `errors.New()` instead of `errors.Errorf()`
2. wordsmithing: "Initializing a project directly within $GOPATH/src is not currently supported"